### ISSUE TITLE
Add dark mode option to reader text

### ISSUE THREAD

**Issue by ZoeKHarvey** [Guten-Reader/guten_reader_FE#57]
* When a user pushes a button, colors on the reader page should invert so font color is white and background is black.
* Good for people with sensitivity to light, some forms of color blindness and makes it less prone to triggering photophobia
